sessionCheck.js

```diff
@@ -147,11 +149,14 @@
             var authorizationUrl = config.opUrl + "?prompt=none" +
                     "&client_id="     + config.clientId +
                     "&response_type=" + config.responseType +
-                    "&redirect_uri="  + config.redirectUri;
+                    "&redirect_uri="  + config.redirectUri + 
+                    "&state=" + config.authId;
 
             if (config.responseType === "id_token") {
                 var nonce = Math.floor(Math.random() * 100000);
-                sessionStorage.setItem("sessionCheckNonce", nonce);
+                var nonceMap = sessionStorage.getItem("sessionCheckNonce") ? JSON.parse(sessionStorage.getItem("sessionCheckNonce")) : {};
```

Comment:
Similar to above - maybe worth changing the key name here too.

Reply:
fixed